lib/player_two.rb

```diff
@@ -0,0 +1,14 @@
+class PlayerTwo  
```

Comment:
Yeah definitely don't need an entirely new class just to hold the separate piece type, but you already planned for that in our next refactor!